DAQ/src/CaloHitsFromDTCEvents_module.cc

```diff
@@ -320,16 +320,27 @@ void art::CaloHitsFromDataDTCEvents::analyze_calorimeter_(
       float time = thisHitPacket.Time + peakIndex * digiSampling_ + timeCalib_[SiPMID];
 
       bool isCaphri = offlineId.crystal().isCaphri();
+      if(!isCaphri) {
+        if(offlineId.crystal().disk() == 0) ++nhits_d0;
+        else                                ++nhits_d1;
+      }
 
       // FIX ME! WE NEED TO CHECK IF TEH PULSE IS SATURATED HERE
       if (((eDep >= hitEDepMin_) || (isCaphri && (eDep >= caphriEDepMin_))) &&
```

Comment:
This logic would be clearer split fully into isCaphri and !isCaphri blocks. As is that flag is tested multiple times. I understand that pre-exists this PR, but adding the intensity info branch makes it worse.